orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/cluster/RollbackClusterStage.java

```diff
@@ -87,6 +87,9 @@ public void taskGraph(@Nonnull Stage stage, @Nonnull TaskNode.Builder builder) {
       context.put("credentials", stageData.credentials);
       context.put("cloudProvider", stageData.cloudProvider);
 
+      // propagate any attributes of the parent stage that are relevant to this rollback
+      context.putAll(propagateParentStageContext(stage.getParent()));
+
```

Comment:
Can't it automatically read these things anyway?

Reply:
Not automatically ... they aren't marked as `outputs`.